### PR TITLE
add code coverage

### PR DESCRIPTION
**Pull request by HarrisChu**
1. https://github.com/vesoft-inc/nebula-dev-docker/pull/57, waiting for this pr merged.
2. need to verify codecov.io no need token via github action.

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -141,6 +142,15 @@ jobs:
           make RM_DIR=false down
         working-directory: tests/
         timeout-minutes: 2
+      - name: coverage
```
Comment (yixinglu) [inline .github/workflows/pull_request.yml:145]: if: ${{ matrix.os }} == "ubuntu2004"

File: .github/workflows/pull_request.yml [hunk 2]
```diff
@@ -141,6 +142,15 @@ jobs:
...
+        run: |
+          case ${{ matrix.os }} in 
+          ubuntu2004)
+            ~/.local/bin/fastcov -d build -l -o fastcov.info -p --exclude /usr/include --exclude=scanner.lex
```
Comment (yixinglu) [inline .github/workflows/pull_request.yml:149]: --exclude=/opt/vesoft/* ?

**Review by yixinglu — DISMISSED** [review]

**Comment by yixinglu** [comment]
@HarrisChu it seems that you shoud install `fastcov` firstly in `vesoft/nebula-dev` image

**Comment by HarrisChu** [comment]
it's ok now, https://codecov.io/gh/vesoft-inc/nebula/tree/f506b6ef68c4e415903496160390e6fe656e3b41

but seems if there's any failed job, it won't display codecov bot in conversation

**Review by yixinglu — APPROVED** [review]

**Review by jievince — APPROVED** [review]

**Pull request merged**